rafthttp/http.go

```diff
@@ -29,6 +29,8 @@ import (
 
 const (
 	ConnReadLimitByte = 64 * 1024
+
+	errMsgIncompatibleVersion = "incompatible version"
```

Comment:
can we define this as error?
so line 163 can return it. and when we want the string we can call err.Error()